docs/open-source-tizen/tools/gbs/gbs-clone.md

```diff
@@ -1,87 +1,75 @@
-                    of target remote repository.
- 
-                    A typical URL mainly contains the following information:
- 
-                    * Transport protocal
-                    * Remote server
-                    * Path to the target remote repository
- 
-                    The syntaxes are shown below:
- 
-                    ssh://<User>@<Remote_Server>[:<Port>]/<Path_to_Target_Remote_Repository>
-                    https://<User>:<HTTP_Password>@<Remote_Server>/gerrit/p/<Path_to_Target_Remote_Repository>
- 
-                    **Note:** For more information about the "<HTTP_Password>",
-                    refer to Section 1 in `Cloning Tizen Source`_.
-```
+- Optional parameters:
 
-### Optional Parameters
+  ```bash
+  -h, --help        Shows the help message and exit.
+  --upstream-branch <Upstream_Branch>
+                    Specifies the upstream branch of the target package when
+                    its upstream does not match the default settings, including
+                    the following:
 
-```bash
--h, --help        Shows the help message and exit.
---upstream-branch <Upstream_Branch>
-                  Specifies the upstream branch of the target package when
-                  its upstream does not match the default settings, including
-                  the following:
- 
-                  * upstream
-                  * master
-                  * pristine-tar
- 
-                  When cloning with this parameter, the upstream branches
-                  cloned to local disk change to the following:
- 
-                  * <Upstream_Branch>
-                  * master
-                  * pristine-tar
- 
---packaging-branch <Working_Branch>
-                  Specifies the working branch that will be checked out.
- 
---all             Tracks all remote branches.
- 
---depth <Depth>   Creates a shallow clone with a history truncated to the
-                  specified number of revisions.
- 
-<Directory>       Specifies the destination directory into which GBS clones
-                  the repository.
-```
+                    * upstream
+                    * master
+                    * pristine-tar
+
+                    When cloning with this parameter, the upstream branches
+                    cloned to local disk change to the following:
+
+                    * <Upstream_Branch>
+                    * master
+                    * pristine-tar
+
+  --packaging-branch <Working_Branch>
+                    Specifies the working branch that will be checked out.
+
+  --all             Tracks all remote branches.
+
+  --depth <Depth>   Creates a shallow clone with a history truncated to the
+                    specified number of revisions.
+
+  <Directory>       Specifies the destination directory into which GBS clones
+                    the repository.
+  ```
 
-## Examples
+Examples of command usage:
 
-- Clone a Tizen package
+- Clone a Tizen package:
```

Comment:
Same as above

Reply:
I do not understand what this refers to.
There is the "Examples of command usage" text here. If you want to change it to  "Examples" in the above comments, we will of course change it here too.

Or do you mean something else?